wemake_python_styleguide/config.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+import configparser
+from pathlib import Path
+from typing import Optional
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+class ConfigFileParser(object):
+    """This class parses configs from `setup.cfg` file."""
+
+    CONFIG_FILE_NAME = 'setup.cfg'
```

Comment:
We should not rely on this value, so it should be removed. We do not take part inn parsing config file ourself, `flake8` does it for us.